### DISCUSSION TITLE
How to scan a remote git repository

### DISCUSSION THREAD

**Discussion by hermanmaleiane** [aquasecurity/trivy#1606] [Q&A] [answered]
Hi team,

Hope you are doing well.
I'm trying to use trivy to scan a remote git repository but i don't know how to start.
There is no examples on how to use it.
Can you please help?

**Comment by knqyf263** [comment, marked as answer]
You can see here.
https://aquasecurity.github.io/trivy/v0.22.0/vulnerability/scanning/git-repository/
